integrity-backend/input-proofmode-signal-examples/3e11cc57daf3bad8375935cad4878123acc8d769551ff90f1b1bb0dc597-meta-content.json

```diff
@@ -32,6 +32,17 @@
       }
     ],
     "private": {
+      "geolocation": {
+        "latitude": "123.456",
+        "longitude": "-123.456",
+        "altitude": "123.456",
+        "timestamp": "2022-08-15T16:24:20.305Z",
+        "city": "Stockton",
+        "countryCode": "us",
+        "countryName": "United States",
+        "provinceState": "California",
+        "address": "123 Fake St, Stockton, California, 20002, United States"
```

Comment:
It's tricky to parse out just "123 Fake St" from the reverse-geocode API for all kinds of different locations. IMO it's better to just put the full address / display name in this field.